calabash/Classes/FranklyServer/Routes/LPScreenshotRoute.m

```diff
@@ -32,7 +32,14 @@ - (BOOL) supportsMethod:(NSString *) method atPath:(NSString *) path {
 
 - (NSData *) takeScreenshot {
 
-  CGSize imageSize = [[UIScreen mainScreen] bounds].size;
+
+  // Available on iOS >= 7
+  SEL selector = @selector(drawViewHierarchyInRect:afterScreenUpdates:);
+  UIView *view = [[UIView alloc] initWithFrame:CGRectZero];
+  BOOL drawViewHierarchy = [view respondsToSelector:selector];
```

Comment:
Screenshotting is slow.  I wanted avoid a extra message for each window.  I suppose there are only going to 4 or so windows max.